device-types/Dell/PowerSwitch-N2248X-ON.yaml

```diff
@@ -0,0 +1,131 @@
+---
+manufacturer: Dell
+model: PowerSwitch N2248X-ON
+slug: dell-powerswitch-n2248x-on
+part_number: N2248X-ON
+u_height: 1
+is_full_depth: true
```

Comment:
Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
Airflow is not listed because there are two types, IO to PSU or PSU to IO.